internal/pkg/cli/delete/static_site_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package delete
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/aws/copilot-cli/internal/pkg/cli/delete/mocks"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStaticSite_CleanResources(t *testing.T) {
+	type moks struct {
```

Comment:
on purpose???
```suggestion
	type mocks struct {
```

Reply:
yeah lol :sob: so it doesn't override the _package_ named `mocks`